sdk/keyvault/Azure.Security.KeyVault.Secrets/tests/SecretClientLiveTests.cs

```diff
@@ -479,5 +480,27 @@ public async Task AuthenticateCrossTenant()
 
             Assert.AreEqual(200, response.GetRawResponse().Status);
         }
+
+        [RecordedTest]
+        public async Task GetNonExistentSecretNoThrow()
```

Comment:
Would it be worth adding a test to validate that errors aren't logged either?

Reply:
As opposed to distributed tracing you mean? If there's some easy way, sure, but I also feel that making sure DT and logging are logged or not should be in the purview of Azure.Core and not repeated throughout. Better to isolate the tests of such conditionals to whatever implements them, which is Core.

Reply:
Sounds good.

Reply:
@annelo-msft do you, @JoshLove-msft, @christothes and whoever else is focused on core feel it has decent general test coverage for dtracing now?